assets/js/cycles.js

```diff
@@ -23,5 +23,25 @@ var Cycles = {
                 .replace('{month}', Language.get(`menu.month.${_date[0]}`))
                 .replace('{day}', _date[1])
         );
+        return _date[1];
     }
-}
\ No newline at end of file
+}
+
+// show alert when cycles are not up to date
+setInterval(function () {
+    var newDate = new Date(),
+        day = newDate.getUTCDate(),
+        hour = newDate.getUTCHours(),
+        minute = newDate.getUTCMinutes();
+    
+    if (hour < 1 && minute <= 20)
```

Comment:
Why this check? As long as it hasn't been updated "today" (in UTC) it's fine, no need to force us to update it in 20 minutes. ;-)

Reply:
ok, that is not needed